keylime/tenant.py

```diff
@@ -864,6 +864,65 @@ def do_verify(self):
                 continue
             break
 
+    def do_add_allowlist(self, name, tpm_policy=None, vtpm_policy=None,
+                         allowlist=None, exclude_list=None):
+
+        _tpm_policy = None
```

Comment:
I don't think we need to prefix these variables with an underscore. They aren't class variables and can't be accessed outside of the method anyway. And it's inconsistent that some variables have this convention while other internal vars (like `al_data`, `excl_data`, etc) don't.